lms/envs/common.py

```diff
@@ -590,6 +590,18 @@
     }
 }
 
+COURSE_MODE_DEFAULTS = {
+    'bulk_sku': None,
+    'currency': 'usd',
+    'description': None,
+    'expiration_datetime': None,
+    'min_price': 0,
+    'name': 'Audit',
```

Comment:
This was previously extracted for internationalization.  We should continue to do that.